app/models/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
```

Comment:
Looks like the idea is to add functionality to that instead of monkey-patching ActiveRecord::Base: http://blog.bigbinary.com/2015/12/28/application-record-in-rails-5.html

But if we switch to that at some point, we should do it separately, for all classes at once.